tests/test_store_policy_resolver.py

```
@@ -0,0 +1,377 @@
+            store_product_uuid = config.policy_uuid
+            config = StorePolicyResolver.resolve_store_item(store_product_uuid)
+
+            assert config.product_id == 101
+            assert config.entitlement_type == "DELAYED_USE"
+            assert config.policy_uuid == store_product_uuid
+            assert config.class_id == test_class["class_id"]
+
+    def test_resolve_store_item_not_found(self, app):
+        """Test PolicyNotFound when UUID doesn't exist."""
+        with app.app_context():
+            with pytest.raises(PolicyNotFound):
```

Ruff (0.16.0)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)